app/jobs/socure_docv_results_job.rb

```diff
@@ -40,4 +55,18 @@ def socure_document_verification_result
       document_capture_session_uuid:,
     ).fetch
   end
+
+  def document_capture_session
+    @document_capture_session ||=
+      DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
+  end
+
+  def rate_limiter
+    return unless document_capture_session
```

Comment:
this seems not needed since we verify the document capture session exist in the constructor